contracts/ColonyNetwork.sol

```diff
@@ -135,6 +135,7 @@ contract ColonyNetwork is ColonyNetworkStorage {
 
     authority.setOwner(etherRouter);
     colony.setOwnerRole(msg.sender);
+    colony.setRecoveryRole(msg.sender);
```

Comment:
Yeah, this is nice, but there's not *actually* any reason to do this here, is there? The owner could do this further down the road for themselves, once they've been educated as to what it means / does.

Reply:
Good point